plugins/modules/rds_cluster.py

```diff
@@ -187,11 +187,16 @@
         description:
           - Specifies the storage type to be associated with the DB cluster.
           - This setting is required to create a Multi-AZ DB cluster.
-          - When specified, a value for the I(iops) parameter is required.
-          - Defaults to C(io1).
+          - For multi-AZ DB clusters, I(storage_type) defaults to C(io1) and a value for the I(iops) parameter is required.
```

Comment:
https://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#semantic-markup-within-module-documentation

```suggestion
          - For multi-AZ DB clusters, O(storage_type) defaults to V(io1) and a value for the V(iops) parameter is required.
```